zerver/actions/message_send.py

```diff
@@ -1277,6 +1277,20 @@ def send_pm_if_empty_stream(
         send_rate_limited_pm_notification_to_bot_owner(sender, realm, content)
 
 
+def send_message_to_reportmsg_stream(
+    sender: UserProfile, reporter: UserProfile, realm: Realm, message: str, topic_name: str
+) -> None:
+    reportmsg_stream = realm.get_reportmsg_stream()
+    if reportmsg_stream is None:
+        # obscure error condition where the stream was deleted after being set as the target.
+        # future: consider defaulting to sending a huddle-message to the owners
```

Comment:
Just to be clear, would this happen due to a race with deactivation or something, or only if someone goes in and deletes the stream from the database manually?

Reply:
DONE - improved comment (db corruption, not a normal error assuming the code works to disallow deactivating the reportmsg stream... I also added a test case for to detect that...)